cmd/dump.go

```diff
@@ -164,6 +170,16 @@ func runDump(ctx *cli.Context) error {
 	return nil
 }
 
+func getDumpFileName(ctx *cli.Context) string {
+	fmtString := ctx.String("file")
+
+	if strings.Count(fmtString, "%d") > 0 {
+		fmtString = fmt.Sprintf(fmtString, time.Now().Unix())
```

Comment:
In reference to my other comment, personally I think this is unnecessary.  
  
If a user wants to override the file name, they should be able to provide the exact name they want, without relying on the code transforming a `%d`. This code would only transform a single `%d` as well, and if they had any other formatting symbols before/after, they would also end up with a bad filename.  
  
I will not block this PR, however I don't personally approve of this "magic".
(To clarify, I _do_ like this PR. Overriding the filename would be a great feature. I just don't particularly care for this part.